WORKSPACE

```diff
@@ -30,7 +30,7 @@ maven_install(
         "org.elasticsearch:elasticsearch:%s" % elasticsearch_version,
         "org.elasticsearch.test:framework:%s" % elasticsearch_version,
         "org.elasticsearch.client:elasticsearch-rest-high-level-client:%s" % elasticsearch_version,
-        'org.apache.logging.log4j:log4j-api:jar:2.11.1',
```

Comment:
Can you update all the log4j artefacts to 2.17 ? Maybe we could have this as a variable too to avoid repetition :)

Reply:
good catch! Yes great idea, I made it a variable in version bzl file